docs/velox-backend-limitations.md

```diff
@@ -47,6 +47,8 @@ In certain cases, Gluten result may be different from Vanilla spark.
 #### JSON functions
 Velox only supports double quotes surrounded strings, not single quotes, in JSON data. If single quotes are used, gluten will produce incorrect result.
 
+Velox doesn't support [*] in path when get_json_object function is called and returns null instead
```

Comment:
Nit: please add "." at the end of statement.

Reply:
Done